### PR TITLE
Remove trailing spaces and fix identation

### PR DESCRIPTION
**Pull request by ylavoie**
All sources files must not have lines with trailing spaces. This PR fixes the few remaining ones.
It also fix identation for the few ones which have escaped previous normalization PR

### CODE REVIEW

File: UI/lib/ui-header.html [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
-        <script type="text/javascript" src="js/dojo/dojo.js"></script>
-        <script type="text/javascript" src="js/lsmb/main.js"></script>
-        [% FOREACH s = include_script %]
-            <script type="text/javascript" src="[% s %]" ></script>
-        [% END %]
-        <meta name="robots" content="noindex,nofollow" />
+    <title>[% title %]</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    [%+# HTML Snippet, for import only %]
+    [%+#
```
Comment (ehuelsmann) [inline UI/lib/ui-header.html:16]: @ylavoie, are the plus-signs you've introduced here accidental or intended? If intended, what's their purpose?

File: t/data/04-complex_template.html [hunk 2]
```diff
@@ -651,4 +651,4 @@
 </div>
 <?lsmb END ?>
 </body>
-</html>
+<?lsmb end_html ?>
```
Comment (ehuelsmann) [inline t/data/04-complex_template.html:654]: hmm. This one belongs in the previous PR I merged, I think. Well, lets leave it that way.

File: UI/lib/ui-header.html (hunk 1 above)
Comment (ylavoie) [reply UI/lib/ui-header.html:16]: White space chomping options. PRE_CHOMP (`[%x`) and POST_CHOMP (`x%]`) where `x` can be set to:
```
Constant      Value   Tag Modifier
----------------------------------
CHOMP_NONE         0          +
CHOMP_ONE            1          -
CHOMP_COLLAPSE  2          =
CHOMP_GREEDY      3          ~
```

File: t/data/04-complex_template.html (hunk 2 above)
Comment (ylavoie) [reply t/data/04-complex_template.html:654]: Missed 1. Sorry

File: UI/lib/ui-header.html (hunk 1 above)
Comment (ehuelsmann) [reply UI/lib/ui-header.html:16]: That warrants a comment of its own, in the source, I'd say...

Comment (ylavoie) [reply UI/lib/ui-header.html:20]: Done

Comment (ehuelsmann) [reply UI/lib/ui-header.html:20]: Thanks.

**Pull request merged**